infra/frontend/app-config/dev.tf

```diff
@@ -15,4 +15,8 @@ module "dev_config" {
   instance_scaling_min_capacity   = 2
   # instance_scaling_max_capacity is 5x the instance_scaling_min_capacity
   instance_scaling_max_capacity = 10
+
+  service_override_extra_environment_variables = {
+    NEW_RELIC_ENABLED = "true"
```

Comment:
Is it intentional that you're using `false` (bool) versus `"true"` string?

Reply:
Oops! That mismatch is definitely a mistake. I wanted to follow the type definition here, so updating the `false` to null. https://github.com/HHS/simpler-grants-gov/blob/e9eed30cfc7a09c7851b0b85b4a4853aba774cd5/infra/frontend/app-config/env-config/variables.tf#L57